settings.example.js

```diff
@@ -1,7 +1,16 @@
 var settings = {
-    username: "develina.devsson@mydevtenant.onmicrosoft.com",
-    password: "pass@word1",
-    siteUrl: "https://mydevtenant.sharepoint.com/" 
+
+    spsave: {
+        username: "develina.devsson@mydevtenant.onmicrosoft.com",
+        password: "pass@word1",
+        siteUrl: "https://mydevtenant.sharepoint.com/"
+    },
+    testing: {
+        clientId: "{ client id }",
```

Comment:
I've added a link to the wiki links about how to Register SharePoint Add-ins 2013

[Register SharePoint Add-ins 2013](https://github.com/OfficeDev/PnP-JS-Core/wiki/Links/_compare/199c73a4e1df0ebe47b95df18b8706e764ad2768#diff-7326242e97244540da1b3e8ca5a7a492R13)